tests/virt_autotest/kubevirt_tests_server.pm

```diff
@@ -203,34 +203,55 @@ sub install_kubevirt_packages {
     my $self = shift;
     # Install required kubevirt packages
     my $os_version = get_var('VERSION');
-    my $virt_tests_repo = get_required_var('VIRT_TESTS_REPO');
-    my $virt_manifests_repo = get_var('VIRT_MANIFESTS_REPO');
+    my $virt_manifests_repo;
+    my $virt_tests_repo;
+    my $virt_manifests_pkgs = 'containerized-data-importer-manifests kubevirt-manifests kubevirt-virtctl';
+    my $virt_tests_pkg = 'kubevirt-tests';
+    my $search_manifests;
 
     record_info('Install kubevirt packages', '');
     # Development Tools repo for OBS Module, e.g. http://download.suse.de/download/ibs/SUSE/Products/SLE-Module-Development-Tools-OBS/15-SP4/x86_64/product/
     # Development product test repo for SLE official product OSD testing, e.g. http://download.suse.de/ibs/SUSE:/SLE-15-SP4:/GA/standard/
     # Devel test repo, e.g. http://download.suse.de/download/ibs/Devel:/Virt:/SLE-15-SP4/SUSE_SLE-15-SP4_Update_standard/
     # MU product test (SLE official MU channel+incidents)
-    transactional::enter_trup_shell(global_options => '--drop-if-no-change') if (is_transactional);
+    if (get_var('INCIDENT_REPO')) {
+        zypper_call("in -f $virt_manifests_pkgs $virt_tests_pkg");
 
-    zypper_call("lr -d");
-    zypper_call("ar $virt_tests_repo Virt-Tests-Repo");
-    zypper_call("ar $virt_manifests_repo Virt-Manifests-Repo") if ($virt_manifests_repo);
-    zypper_call("--gpg-auto-import-keys ref");
-
-    my $virt_manifests = 'containerized-data-importer-manifests kubevirt-manifests kubevirt-virtctl';
-    my $search_manifests = $virt_manifests =~ s/\s+/\\\|/gr;
+        # Check if at least one installed kubevirt package is from the incident repo
+        my $pkgs_from_incident_repo;
+        foreach (split(' ', $virt_manifests_pkgs), $virt_tests_pkg) {
+            $pkgs_from_incident_repo += 1 if (script_output("zypper info $_ | awk -F': ' '/^Repository/{print \$2}'") =~ /^TEST_/);
+        }
 
-    if ($virt_manifests_repo) {
-        zypper_call("in -f -r Virt-Manifests-Repo $virt_manifests");
-    } elsif (script_run("rpmquery $virt_manifests")) {
-        if (is_transactional || script_run("zypper se -r SLE-Module-Containers${os_version}-Updates $virt_manifests | grep -w '$search_manifests'")) {
-            zypper_call("in -f $virt_manifests");
+        if ($pkgs_from_incident_repo < 1) {
+            die "No kubevirt packages were installed from incident repositary.";
         } else {
-            zypper_call("in -f -r SLE-Module-Containers${os_version}-Updates $virt_manifests");
+            record_info("$pkgs_from_incident_repo package(s) installed from incident repositary.", '');
```

Comment:
I do not find this record_info in the validation job. Would you please share the link for it?

Reply:
I made a small change for the output of the text info, will update in a new validation run.
https://openqa.suse.de/tests/18721076#step/kubevirt_tests_server/43

Reply:
Record info in the latest run: https://openqa.suse.de/tests/18746145#step/kubevirt_tests_server/43

Reply:
OK, there are already two places for displaying the kubevirt package info. Anyway I will update it.
https://openqa.suse.de/tests/18746145#step/kubevirt_tests_server/29
https://openqa.suse.de/tests/18746145#step/kubevirt_tests_server/61

Reply:
It will be good to merge IMO after this change. Thanks!

Reply:
Cool! Wait for the final validation link!

Reply:
Looks good,thanks.